frontend/ui/lib/widgets/trip_view_widget.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/place_block_widget.dart';
+
+class TripViewWidget extends StatefulWidget {
+  TripViewWidget({Key key}) : super(key: key);
+  TripWidgetState createState() => TripWidgetState();
+}
+
+class TripWidgetState extends State<TripViewWidget> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+          border: Border.all(
+            color: Colors.black,
+          ),
+          borderRadius: BorderRadius.all(Radius.circular(5))),
+      margin: const EdgeInsets.all(15.0),
+      padding: const EdgeInsets.all(10.0),
+      width: MediaQuery.of(context).size.width * .80,
+      child: ExpansionTile(
+        title: Text("Trip Name"),
+        trailing: Text("M T D"),
```

Comment:
Since these were going to need a bunch of functionality added I was going to make them into buttons a separate time and really focusing in on the look. Didn't want to do too much in this PR.